README.md

```diff
@@ -5,4 +5,6 @@ description
 ## Subheader
 
 Trying new things
---Commit and push
\ No newline at end of file
+--Commit and push
+Added new feature
+This should appear in main branch after Merge
```

Comment:
This is the main thing I want to see once this feature branch gets merged to the main branch.